sds_data_manager/stacks/ialirt_ingest_lambda_stack.py

```diff
@@ -0,0 +1,158 @@
+"""Configure the ialirt ingest lambda stack."""
+
+import pathlib
+
+import aws_cdk as cdk
+from aws_cdk import RemovalPolicy, Stack, aws_dynamodb, aws_s3
+from aws_cdk import aws_dynamodb as ddb
+from aws_cdk import aws_events as events
+from aws_cdk import aws_events_targets as targets
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as lambda_
+from aws_cdk import aws_lambda_python_alpha as lambda_alpha_
+from constructs import Construct
+
+
+class IalirtIngestLambda(Stack):
+    """Stack for ialirt ingest lambda."""
+
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        env: cdk.Environment,
+        ialirt_bucket: aws_s3.Bucket,
+        **kwargs,
+    ) -> None:
+        """IalirtIngestLambda Stack.
+
+        Parameters
+        ----------
+        scope : Construct
+            Parent construct.
+        construct_id : str
+            A unique string identifier for this construct.
+        env : obj
+            Environment.
+        ialirt_bucket : aws_s3.Bucket
+            The data bucket.
+        kwargs : dict
+            Keyword arguments.
+
+        """
+        super().__init__(scope, construct_id, env=env, **kwargs)
+
+        # Create DynamoDB Table
+        self.packet_data_table = self.create_dynamodb_table()
+
+        # Create Lambda Function
+        self.ialirt_ingest_lambda = self.create_lambda_function(
+            ialirt_bucket, self.packet_data_table
+        )
+
+        # Create Event Rule
+        self.create_event_rule(ialirt_bucket, self.ialirt_ingest_lambda)
+
+    def create_dynamodb_table(self) -> aws_dynamodb.Table:
```

Comment:
Why do we have two Tables being created? We have this one here, but we also have the one in `ialirt_database_stack` As far as I can tell the setup looks identical.

Reply:
Yes. I meant to delete that.